backend/apis/google-natural-lang-api.ts

```diff
@@ -0,0 +1,62 @@
+import * as path from 'path';
+import dotenv from 'dotenv';
+dotenv.config();
+
+
+const filePath = path.resolve(__dirname, './.gac.json');
+process.env.GOOGLE_APPLICATION_CREDENTIALS = filePath;
+
+import { LanguageServiceClient } from '@google-cloud/language';
+
+const languageClient = new LanguageServiceClient();
+
+const callClassifyText = async function(inputText: string) {
+  
+  const document: IDocument = {
+    content: inputText,
+    type: 'PLAIN_TEXT', // or 'HTML' for HTML content
+  };
+
+  interface IDocument {
+    content: string;
+    type: "PLAIN_TEXT" | "HTML" | "TYPE_UNSPECIFIED"; // Ensure it matches the expected values/types
+    // Other properties...
+  }
+
+  // Detects sentiment of entities in the document
+  const [result] = await languageClient.analyzeEntitySentiment({ document });
```

Comment:
can we surround the await statement in a try catch block?

I think this (the code you have below) handles uncaught errors thrown, but it lives in the process so it's global.  Kinda weird to catch all exceptions in the application that lives in a google API file.

```js
process.on('unhandledRejection', err => {
  console.error(err);
  process.exitCode = 1;
});
```